### ISSUE TITLE
Some text in the .uexp file cannot be read.

### ISSUE THREAD

**Issue by BeanGTA** [amrshaheen61/UE4LocalizationsTool#45]
<a href="https://imgbb.com/"><img src="https://i.ibb.co/Gn6qy4N/image.png" alt="image" border="0"></a>

Link File: https://mega.nz/folder/5rhkXBIb#SBmOXLJ9a1kUVa4plm951g

**Comment by amrshaheen61** [comment]
@BeanGTA 
you have another example?

**Comment by BeanGTA** [comment]
> @BeanGTA you have another example?

That subtitle file is the game from UE3 upgraded to UE4. That game is called The Last Remnant. In that file there is a section called .RushMessageSource, I guess that is the UE plugin. But when I check in UE4 to search for class . RushMessageSource doesn't exist, even in UE3. Maybe that was made by the game manufacturer RushMessageSource. I tried your software to run other files and it worked fine. But when using the file I sent you the link to, the text is not displayed. I'm not a Hex expert, I've only been learning Hex for 2 weeks. That's why I reported the bug.
